src/Components/login.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Image from "./dashboardRandomImage";
+
+const Login = ({ loading, products }) => {
+  const [person, setPerson] = useState({ name: "", email: "", password: "" });
+  let navigate = useNavigate();
+
+  function ValidateEmail(mail) {
+    if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(mail)) {
+      return true;
+    }
+    return false;
+  }
+
+  const handleSubmit = (e) => {
```

Comment:
handlesubmit should perform submit operation. Pull out validation part in diffent func